pkg/cmd/render/render.go

```diff
@@ -162,6 +169,22 @@ func (r *renderOpts) Run() error {
 			return fmt.Errorf("unable to parse restricted CIDRs from config %q: %v", r.clusterConfigFile, err)
 		}
 	}
+	if len(r.clusterAuthFile) > 0 {
+		clusterAuthFileData, err := ioutil.ReadFile(r.clusterAuthFile)
+		if err != nil && !os.IsNotExist(err) {
```

Comment:
ignoring a not found for a flag that is explicitly set should at least log what happened